groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovyTextDocumentService.java

```diff
@@ -112,14 +116,13 @@ public GroovyTextDocumentService(CompilerWrapperProvider provider) {
 
     @Override
     public CompletableFuture<List<? extends SymbolInformation>> documentSymbol(DocumentSymbolParams params) {
-        String uri = params.getTextDocument().getUri();
-        Path filePath = Paths.get(uri);
-        if (!filePath.isAbsolute()) {
-            uri = provider.get().getWorkspaceRoot().resolve(uri).toAbsolutePath().toString();
+        Path absoluteUri = getAbsolutePathRelativeToWorkspace(params.getTextDocument().getUri());
+        if (!absoluteUri.toFile().exists()) {
+            log.warn("Calling documentSymbol on non-existing uri {}", absoluteUri.toString());
```

Comment:
I decided to just keep going (and return an empty list) because I thought it might be better to only throw an exception if there's no alternative, like the method absolutely requires it to exist (like in the case of didChange, didClose, didSave). Not sure what the right approach is, thoughts?